README.md

```diff
@@ -85,3 +85,34 @@ This configuration will only send multicast packets to the interface assigned
 the `192.168.1.1` address. The interface lookup is dynamic each time an mDNS
 query is sent, so if the address moves to a different interface the plugin
 will automatically switch to the new one.
+
+## Service Discovery
+
+Queries to mDNS are constrained to the `local` domain, alternative domains that an mDNS server publishes are not supported by this plugin. This does not restrict you to `.local` TLD, as the address is referenced from the service responses belonging to the `local` domain.
+
+If you have multiple network interfaces that respond to mDNS for your host(eg on the same system that CoreDNS is running), this can result in the wrong IP address returned for a hostname due to a race condition with multicast. Make sure you configure your mDNS server to whitelist the network interface with the assigned IP address you want to associate the hostname to.
+
+If a query is not responding, check the log for hosts discovered by this plugin reported as `mdnsHosts`. The plugin will populate `mdnsHosts` by **only discovering** mDNS services of the type `_workstation._tcp`.
```

Comment:
Just a note that it does also recognize services of the type _etcd-server-ssl._tcp because of a previous requirement that we had, but that's fairly specific to our use case.

I should probably update the docs above to specify that that is the SRV record referred to by minimum SRV records.

Reply:
I did notice that I think, but I wasn't sure how to document it or if it mattered much. If the type of service(s) to be discovered is to be restricted, then regardless of the what the service would be classed under, they will need to publish the service under what the plugin discovers to be compatible with this plugin. 

Most of the time I assume that's going to be for identifying machines so just keeping it simple by only mentioning the `_workstation._tcp` service made sense? This was my usecase (sub-labels for hosted services via reverse proxy aside) and it wasn't immediately obvious to me why the plugin wasn't discovering some published hostnames.

Reply:
This actually confused me today quite a bit as well. The documentation talks about mDNS all the time, but what it really is doing is using DNS-SD (service discovery) to gather SRV records and relying them. While DNS-SD records can be transmitted over mDNS this is not the same thing as a plain mDNS lookup...

Reply:
Just to wrap this up, I'm good with only mentioning the workstation record. I actually have a patch written somewhere to remove the SRV lookup since we don't need it anymore and it's just generating unnecessary traffic. I'll try to get that in soon.